### PR TITLE
Enhancements on UX flows section 

### PR DESCRIPTION
**Pull request by duyguHsnHsn**
What: Added some analysis  and mockups on the UX flows section, which evaluates 2 variants of working with Python steps in  Jupyter. Also fixed some typos.

### CODE REVIEW

File: specs/vep-994-jupyter-notebook-integration/README.md [hunk 1]
```diff
@@ -73,29 +73,72 @@ Legend (terms and table are based on Pragmatic Marketing recommendations):
...
+| Error preventions                                                                                         | 1                                                                                                                                    | 1                                                                                                                                                   |
+| Ease of learning: how fast a user who has never seen the user interface before can accomplish basic tasks | 1 <br /> it is harder since the magic commands and writing a function to multiple cells is Jupyter specific                          | 2 <br />  calling functions from one function is a thing we do generally in coding                                                                  |
+
+    Solution 2 total score: 18       Solution 1 total score: 23
+Final decision: Solution 1
```
Comment (tozka) [inline specs/vep-994-jupyter-notebook-integration/README.md:141]: Mention that both solutions are not Mutually Exclusive. That is at some point we can implement both. But for the first iteration/version based on above analysis we will go with Solution 1.

**Review by tozka — APPROVED** [review]

File: specs/vep-994-jupyter-notebook-integration/README.md [hunk 2]
```diff
@@ -73,29 +73,72 @@ Legend (terms and table are based on Pragmatic Marketing recommendations):
...
+|       Delete job       |                                                                                                                                                                                                  "Delete job" will be introduced just like "Create job".                                                                                                                                                                                                  |         1          |   mid    |
+|  Work with SQL steps   |                                                                   One SQL step can be introduced as one cell. The cells that will be part of the job should have a tag ("vdk", "vdk-sql") that shows that the cell should be included in the job. All the cells that does not have that tag will not be included in he job as a step. One notebook file can have as several SQL steps.                                                                    |      2,3,4,7       |   high   |
```
Comment (doks5) [inline specs/vep-994-jupyter-notebook-integration/README.md:99]:
```suggestion
|  Work with SQL steps   |                                                                   One SQL step can be introduced as one cell. The cells that will be part of the job should have a tag ("vdk", "vdk-sql") that shows that the cell should be included in the job. All the cells that do not have that tag will not be included in the job as a step. One notebook file can have several SQL steps.                                                                    |      2,3,4,7       |   high   |
```

**Review by doks5 — APPROVED** [review]

File: specs/vep-994-jupyter-notebook-integration/README.md [hunk 3]
```diff
@@ -129,16 +176,25 @@ Example use case:
...
+The availability of the extension will be managed by JupyterLab since it is going to be run as part of the JupyterLab ecosystem.
 
+### Test Plan
+* The front-end extension is using Jest for JavaScript code testing.
+* This extension uses Playwright for the integration tests (aka user level tests). More precisely, the JupyterLab helper Galata is used to handle testing the extension in JupyterLab.
+* The server extension is using Pytest for Python code testing.
+* The VDK notebook plugin will be using Pytest for Python code testing.
+
+### Security and Permissions
+In terms of security, Jupyter uses tornado to ensure only authorized user can request the Jupyter server
```
Comment (tozka) [inline specs/vep-994-jupyter-notebook-integration/README.md:189]: You can add a todo only but a few more security concerns need to be looked into 
* What would be the authenticaiton/authorizaiton against the control service  -you can refer to control service doc for this.

File: specs/vep-994-jupyter-notebook-integration/README.md [hunk 4]
```diff
@@ -129,16 +176,25 @@ Example use case:
...
+You can read more [here](https://jupyterlab-server.readthedocs.io/en/stable/api/index.html).
```
Comment (tozka) [inline specs/vep-994-jupyter-notebook-integration/README.md:190]: Making sure to supply references is very important. But in this case I think https://jupyter-notebook.readthedocs.io/en/stable/security.html# is more precise reference for this section.

**Review by tozka — APPROVED** [review]

**Pull request merged**